source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp

```diff
@@ -1010,14 +1010,10 @@ namespace RendererQml
         uiDateInput->Property("font.pixelSize", std::to_string(fontSize));
         uiDateInput->Property("color", context->GetColor(AdaptiveCards::ForegroundColor::Default, false, false));
 
-        if (!input->GetValue().empty())
-        {
-            uiDateInput->Property("text", "\"" + Utils::GetDate(input->GetValue(), false) + "\"");
-        }
-
         uiDateInput->Property("property string selectedDate", "\"" + input->GetValue() + "\"");
+		uiDateInput->AddFunctions(Formatter() << "signal on_" << uiDateInput->GetId() << "TextChanged(var dateText)");
```

Comment:
ideally, signal does not start as on*... 
```suggestion
		uiDateInput->AddFunctions(Formatter() << "signal " << uiDateInput->GetId() << "TextChanged(var dateText)");
```
```suggestion
		uiDateInput->AddFunctions(Formatter() << "signal on_" << uiDateInput->GetId() << "TextChanged(var dateText)");
```

Reply:
Thanks for the feedback. I've made the change with the latest commit